messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerControlWriteHandler.java

```diff
@@ -70,6 +72,21 @@ protected boolean applyUpdateToRuntime(final OperationContext context, final Mod
                                            final ModelNode newValue, final ModelNode currentValue,
                                            final HandbackHolder<Void> handbackHolder) throws OperationFailedException {
         AttributeDefinition attr = getAttributeDefinition(attributeName);
+        if(attr == ServerDefinition.JOURNAL_FILE_SIZE){
```

Comment:
NIOSequentialFileFactory.getAlignment() returns 1, if it's ok to do same constraint 512 for NIO, I'll simply move this in a validator in attribute definition.